OMCompiler/README-Windows-WSL.md

```diff
@@ -0,0 +1,60 @@
+# Windows Subsystem for Linux - README for OpenModelica
+
+This way of building OpenModelica is useful if you want to use a Unix shell and tools on a Windows 10 computer. You don't need to run a VM and Windows Subsystem for Linux (WSL) should be noticeably faster than doing so.
+
+If you want to use the executables from Windows look at [Windows instructions](README-OMDev-MINGW.md) or cross compile for Windows 10 . Please let us know how you did it when it's working ;-)
+
+## Get Windows Subsystem for Linux on Windows 10
+Follow the instructions from Microsoft to install a Linux distribution.
+  - [WSL install guide](https://docs.microsoft.com/de-de/windows/wsl/install-win10)
+
+## Install build dependencies
+See the Dependencies for Linux in [Linux instructions](README.Linux.md) or use
+```bash
+echo deb http://build.openmodelica.org/apt bionic nightly | sudo tee -a /etc/apt/sources.list
+echo deb-src http://build.openmodelica.org/apt bionic nightly | sudo tee -a /etc/apt/sources.list
+
+# Get key for OpenModelica
+# apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 0x3a59b53664970947   # Since there is a bug in WSL with $ apt-key adv
+curl -sL "http://keyserver.ubuntu.com/pks/lookup?op=get&search=0x3A59B53664970947" | sudo apt-key add
```

Comment:
Is there a better way to get the key for OpenModelica for `apt-get update`?

Reply:
`wget -q http://build.openmodelica.org/apt/openmodelica.asc -O- | sudo apt-key add -`